src/renderer/components/screens/AccountDeletionScreen/styles.module.scss

```diff
@@ -0,0 +1,60 @@
+.AccountDeletionScreen {
```

Comment:
Let's keep all class names in lowerCamelCase (https://github.com/deltachat/deltachat-desktop/blob/master/docs/STYLES.md#naming), this needs to be adjusted for all other classes in this PR as well